### PR TITLE
HOTFIXES

### PR DESCRIPTION
**Pull request by DeadLineSS13**
## This pr solve problem with:
- [ ] OOC chat **(need test)**
- [x]  DOOC chat
- [x] LOOC chat
- [x]  AOOC chat
- [ ]   Solve problem with modes **(need test)**
- [x]   Solve problem with mob health (limbs and etc)

### CODE REVIEW

File: code/controllers/configuration/entries/event.dm [hunk 1]
```diff
@@ -10,34 +10,28 @@
...
-    config_entry_value = TRUE
 
 /datum/config_entry/flag/looc_allowed
-    config_entry_value = TRUE
 
 /datum/config_entry/flag/dooc_allowed
-    config_entry_value = TRUE
 
 /datum/config_entry/flag/dsay_allowed
-    config_entry_value = TRUE
```
Comment (NanakoAC) [inline code/controllers/configuration/entries/event.dm:25]: why are these removed

Comment (DeadLineSS13) [reply code/controllers/configuration/entries/event.dm:25]: > why are these removed

in config game_options ready

**Review by NanakoAC — APPROVED** [review]
